src/tslint.json

```diff
@@ -0,0 +1,92 @@
+{
+  "rules": {
+    "align": [false, "parameters", "arguments", "statements"],
+    "class-name": true,
+    "comment-format": [true, "check-space"],
+    "curly": true,
+    "eofline": true,
+    "forin": false,
+    "indent": [true, "spaces"],
+    "interface-name": true,
+    "jsdoc-format": false,
+    "label-position": false,
+    "label-undefined": false,
+    "max-line-length": false,
+    "member-access": true,
+    "member-ordering": [true,
+        "public-before-private",
+        "static-before-instance",
+        "variables-before-functions"
+    ],
+    "no-any": false,
+    "no-arg": true,
+    "no-bitwise": true,
+    "no-console": [true,
+      "debug",
+      "info",
+      "log",
+      "time",
+      "error",
+      "warn",
+      "timeEnd",
+      "trace"
+    ],
+    "no-consecutive-blank-lines": true,
+    "no-construct": true,
+    "no-construct-vars": true,
+    "no-debugger": true,
+    "no-duplicate-key": true,
+    "no-duplicate-variable": true,
+    "no-shadowed-variable": true,
+    "no-empty": false,
+    "no-eval": true,
+    "no-internal-module": false,
+    "no-require-imports": false,
+    "no-string-literal": false,
+    "no-switch-case-fall-through": true,
+    "no-trailing-comma": true,
+    "no-trailing-whitespace": true,
+    "no-unreachable": true,
+    "no-unused-expression": true,
+    "no-unused-variable": [false, "check-parameters"],
+    "no-use-before-declare": true,
+    "no-var-keyword": false,
+    "no-var-requires": false,
+    "one-line": [true,
+      "check-open-brace",
+      "check-catch",
+      "check-else",
+      "check-whitespace"
+    ],
+    "quotemark": false,
```

Comment:
yes we can enable this.. for now I want to enable what we have in current tslint.json